web/modules/custom/joinup_document/config/install/node.type.document.yml

```diff
@@ -14,3 +14,6 @@ help: ''
 new_revision: true
 preview_mode: 0
 display_submitted: false
+label_singular: document
+label_plural: documents
+label_count: "Document\x03Documents"
```

Comment:
No, it's correct, as we use the plurals for the facets and the facets are already adding the count, in both contexts: as tabs or as normal facets (under "more"). Also, it's not possible to add the `@count` to the facet label because the label is marku, not plain text and, depending, on the context (tab or 'more' item) the markup is different.

Reply:
OK so we are "hacking" the count labels so we can reuse them in custom templates. This might work fine now but it is not the right solution. We might need these count labels in other places, and especially if the upstream patch lands then we can expect that core + contrib will start using this count label. So we will have to change this at some point anyway, it would be better to do it now that it is all fresh in our minds :)

Maybe we could make a patch for Facets to use these new labels which they can use once the core issue is merged?

Reply:
The solution is to store them as 3rd party settings, in behalf of `joinup_core`, in each bundle. Then we drop the d.o patch because we would not need it anymore, but this has the drawback that we have to implement our own `::getCountLabel()`.

Reply:
Right now I see only to options to fix this:

1. We use the d.o patch with the labels as in the patch. Pros: is a straight fix, Cons: cannot reuse the labels for a potential future “@count items” variant.
1. We don’t use the d.o patch. Instead we store the custom plural variants as  3rd party settings in each bundle entity. Pro: don’t affect potential future  “@count items” variants, Cons: We need or own API. We have to reimplement our custom methods like `::getCountLabel()`

Which one?